databricks_cli/sdk/api_client.py

```diff
@@ -62,16 +62,8 @@ class ApiClient(object):
     A partial Python implementation of dbc rest api
     to be used by different versions of the client.
     """
-    def __init__(self, user = None, password = None, host = None, token = None, configUrl = None,
-            apiVersion = version.API_VERSION, default_headers = {}, verify = True):
-        if configUrl:
```

Comment:
Why we no longer need this block?

Reply:
Turns out we just never used this piece of code. It's even a bit buggy. (credentials isn't defined).